src/types/widget.ts

```diff
@@ -386,6 +386,41 @@ export type Widget<
     | 'ais.places'
     | 'ais.poweredBy'
     | 'ais.queryRules'
+    // @MAJOR: remove individual types for rangeSlider & rangeInput
```

Comment:
Why is it a breaking change to change these to `ais.range`?

https://github.com/algolia/instantsearch.js/blob/27b959cb3997a79a528c1fdd73a84ce1bb9a6f9f/src/widgets/range-slider/range-slider.js#L153

Reply:
not a breaking change, you're right, rather a todo. It required a significant change in the check ui state code to test both  type and widgetType, which is why I'm not doing it here yet. I can change it to another comment if you prefer?

Reply:
Yes, that's less misleading.

Reply:
```suggestion
    // @TODO: remove individual types for rangeSlider & rangeInput once updating checkIndexUiState
```